docs/guides/flux-inspired-practice.md

```diff
@@ -3,27 +3,47 @@ title: Flux inspired practice
 nav: 5
 ---
 
-Although zustand is an unopinionated library, here are some patterns we recommend:
+Although Zustand is an unopinionated library, we do recommend a few patterns.
+These are inspired by practices originally found in [Flux](https://github.com/facebookarchive/flux),
+and more recently [Redux](https://redux.js.org/understanding/thinking-in-redux/three-principles),
+so if you are coming from another library, you should feel right at home.
 
-- Create a single store;
-- Always use `set` to define a store;
-- Define your dispatch functions at the root level of the store to update one or more store slices.
+However, Zustand does differ in some fundamental ways,
+so some terminology may not perfectly align to other libraries.
+
+## Recommended patterns
+
+### Single store
+
+Your applications global state should be located in a single Zustand store.
+
+If you have a large application, Zustand supports [splitting the store into slices](./slices-pattern).
+
+### Use `set` / `setState` to update the store
+
+Always use `set` (or `setState`) to perform updates to your store.
+`set` (and `setState`) ensures the described update is immutably performed to the store.
```

Comment:
Immutability isn't guaranteed by `set/setState`. Developers has to guarantee it.

Reply:
Ah yes, good point! I have updated in [additional commit](https://github.com/pmndrs/zustand/pull/1874/commits/f252b54dd7143172132c9518cb798547b324e038)

Reply:
thanks!